docs/source/mods/metromap.rst

```diff
@@ -0,0 +1,511 @@
+stations. A schematic graph drawing aims to ease navigation, such as how to get
+from A to B in a transportation network. There are several different
+requirements for a schematic graph drawing. In this OptiMod, we will consider an
+octilinear representation, i.e., the edges of the network are restricted to
+horizontals, verticals, and diagonals at 45 degrees. Such a schematic drawing
+is of special interest when showing metro lines, but could also be useful for general
+transportation networks or graphs. Note that an octilinear representation
+requires a maximum vertex degree of eight. If satisfied, this OptiMod computes
+an octilinear representation for any undirected graph. The original geographical
+information of the graph is included. More precisely, the geographical
+information defines an input embedding, which is preserved in the schematic
+representation. The mathematical formulation is based on the one given by
+Nöllenburg :footcite:p:`noellenburg05`. This diploma thesis provides additional
+detailed descriptions and explanations. However, we slightly changed some
+variable types and, hence, some constraints.
+
+
+Problem Specification
+---------------------
+
+Let :math:`G` be an undirected graph with a set of nodes :math:`V` and edges
+:math:`E`. For each node, a geographical position is given as x- and y-coordinates.
+
+In the octilinear representation, the edge can be drawn in eight different
+directions when fixing one of its endpoints. Considering the edge
+:math:`(u,v)\in E`, the node :math:`v` can be in the east, north-east, north,
+north-west, west, south-west, south, or south-east of :math:`u`. We enumerate
+these directions from 0 to 7, starting with the east and traversing the
+directions counter-clockwise. The figure below shows the idea. Assume the node
+:math:`u` is positioned in the center. The edge :math:`(u,v)` can follow the
+directions 0 to 7 that represent the cardinal directions.
+
+
+.. figure:: figures/metromap_directions.png
+    :width: 500
+    :align: center
+    :alt: metromap_directions
+
+    Idea of the octilinear representation. Assuming a node :math:`u` in the
+    center, an edge :math:`(u,v)` can be drawn in directions 0 to 7.
+
+Based on the directions, we can determine conditions on the coordinates. For
+example, if the direction of :math:`(u,v)` is 0, then :math:`u` and :math:`v`
+have the same y-coordinate while the x-coordinate of :math:`v` must be at least
+one larger than the one of :math:`u`.
+
+In this OptiMod, an octilinear graph representation is computed. More precisely,
+for each edge, a direction (0 to 7) is computed, and all nodes are assigned to x-
+and y-coordinates that support the edge directions. Additionally, the following
+is considered
+
+* The original positions of the nodes are respected in the following way:
+
+  * Only directions that align with the original direction or are
+    directly neighbored are allowed. For example, if v is north-east (1) from u in the original
```

Comment:
Missed math notation here for $u$ and $v$.

Reply:
Done